src/vcpkg/install.cpp

```diff
@@ -128,7 +128,7 @@ namespace vcpkg::Install
                 continue;
             }
 
-            const auto suffix = file.generic_u8string().substr(prefix_length + 1);
+            const auto suffix = StringView{file.generic_u8string()}.substr(prefix_length + 1);
```

Comment:
You turned this into a use-after-free because the StringView is wrapping a prvalue.

Reply:
Sorry, should have seen that.